core/templates/dev/head/css/oppia.css

```diff
@@ -3789,12 +3789,13 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-activity-summary-tile .activity-title {
+  word-break: break-all;
   bottom: 0;
   color: white;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
   font-size: 1em;
   line-height: 1.2em;
-  margin: 12px;
+  margin: 0.2em;
```

Comment:
Because this is making the left/right margins very squashed, it might be best to define it to just the top/down margins (see screenshot below).

![image](https://cloud.githubusercontent.com/assets/5758256/22321416/7f5961d8-e349-11e6-821a-4a413d70f4b3.png)